libs/datamodel/core/tests/validation/attributes/schema/on_sqlite.prisma

```diff
@@ -0,0 +1,23 @@
+datasource testds {
+    provider = "sqlite"
+    url = env("TEST_DATABASE_URL")
+    schemas = ["public", "sphere"]
```

Comment:
Should usage of `schemas` not be invalid here?
(And hence also `@@schema` also not allowed to be used in general?)

Reply:
Yes, I think the message isn't clear so this and your previous comment are misinterpreting the error. "@@schema is not defined on the current datasource provider" means you aren't allowed to use `@@schema` at all on the current provider (sqlite on this case), not that the schema isn't defined in the datasource. I'll reformulate that error to make it clearer. You can see the error I think you are thinking about in `non_existing_schema.prisma`.

Reply:
So I think we're good on the validation here, just not how it's phrased. We don't allow specifying the schema property on sqlite. I am now realizing we should have a second error on the datasource, we are only returning an error on the model `@@schema` in this test.

Reply:
Here is a suggestion on how to improve the existing message: https://github.com/prisma/prisma-engines/pull/3123#discussion_r945837963